volatility3/framework/symbols/windows/extensions/pool.py

```diff
@@ -326,12 +326,18 @@ class ExecutiveObject(interfaces.objects.ObjectInterface):
     """This is used as a "mixin" that provides all kernel executive objects
     with a means of finding their own object header."""
 
-    def get_object_header(self) -> "OBJECT_HEADER":
+    def get_object_header(
+        self, symbol_table_name: Optional[str] = None
```

Comment:
Technically an additive change, but I think there's enough gone on in this PR that we could probably do with a MINOR version bump of the framework anyway please.  At least until #1647 gets resolved.